### DISCUSSION TITLE
Does anyone have syntax highlighting or an IntelliJ plugin for the schema language?

### DISCUSSION THREAD

**Discussion by mikehearn** [capnproto/capnproto#2002] [Q&A] [answered]
I found an obsolete/abandoned capnp IntelliJ plugin, which I guess could be updated. But I wonder what other people use to write schema files.

**Comment by kentonv** [comment]
I use vscode with the vscode-capnp extension. Unfortunately the ecosystem here is pretty thin.

**Comment by as-com** [comment, marked as answer]
I use the TextMate bundle with IntelliJ: https://github.com/textmate/capnproto.tmbundle

Autocomplete doesn't work very well though.

**Reply by mikehearn** [reply]
Thanks, this will do for now.
